.github/ISSUE_TEMPLATE/governance-review-request.yaml

```diff
@@ -1,158 +1,166 @@
-name: Governance review request
-description: Ask for a governance review for your project from TAG Contributor Strategy
```

Comment:
change description to "OLD: please file a review request in the TOC repo instead"